src/features/signup/components/faqList.tsx

```diff
@@ -0,0 +1,38 @@
+import { FAQItem } from "./faqItem";
+
+export const FAQList = () => {
+  return (
+    <div className="mt-28 flex flex-col items-center justify-center">
+      <h1 className="mb-8">FAQ</h1>
+      <FAQItem
+        title="what is the focus of this round?"
+        description="This found is focused on supporting Mexican projects working on reducing the gender gap, collaborating with local communities, and ReFi."
```

Comment:
let's keep it general. "This round is focused on supporting public goods projects".
I assume projects forking the projects will add their own faqs + changes these answers